android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -1785,7 +1784,8 @@ private void enableSpeedreaderMode() {
         }
     }
 
-    private void openBraveLeo() {
+    public void openBraveLeo() {
+        BraveLeoUtils.verifySubscription(null);
```

Comment:
just Curious, what's the usecase for using `verifySubscription` here ?

Reply:
it's to figure out do we have a subscription active or not when we open the ai chat. As a user could cancel it from playstore.